tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/ContextManager.scala

```diff
@@ -0,0 +1,21 @@
+package me.shadaj.scalapy.tensorflow
+
+import me.shadaj.scalapy.py
+
+object ContextManager {
+
+ def withContext[T <: Context, R](ctx: T)(function: T => R): R = {
+   ctx.__enter__()
+   val r = function(ctx)
```

Comment:
So... what happens now when a Python exception is thrown from `function`?

Reply:
well all exceptions come from Cpython we can catch them I think but what would you plan to do with them?
Or do you just want a 
```
try{
}finally{
}
```

Reply:
Yeah, I just meant adding `finally`; catching them makes no sense

Reply:
ok